backend/pyproject.toml

```diff
@@ -53,7 +53,8 @@ optional = true
 [tool.poetry.group.tests.dependencies]
 openai = "^1.11.1"
 langchain = "^0.1.5"
-llama-index = "^0.9.15"
+llama-index = "^0.10.15"
+llama-index-core = "^0.10.15"
```

Comment:
Do we need to install `llama-index-core` for the tests to work? It might be already included in `llama-index`, right?